options/options_helper.cc

```diff
@@ -676,6 +774,13 @@ Status GetStringFromStruct(
   return Status::OK();
 }
 
+Status GetStringFromMutableDBOptions(const ConfigOptions& config_options,
```

Comment:
If we provide GetStringFromMutableDBOptions, I think it is natural to provide GetStringFromImmutableDBOptions. Otherwise, we do not need the interface, we can directly call GetStringFromStruct when we need to serialize the mutable or immutable options. Similar for GetStringFromMutableCFOptions. What do you think?

Reply:
The GetStringFromMutable is only required for tests.  Also, the GetStringFromStruct will be going away in a future PR